src/main/java/Jane.java

```diff
@@ -0,0 +1,269 @@
+            }
+            return String.format("E|%d| %s |%s|%s", i, this.description, this.from, this.to );
+        }
+        @Override
+        public String toString() {
+            String first = DateTimeFormatter.ofLocalizedDate(FormatStyle.FULL).format(from);
+            String end = DateTimeFormatter.ofLocalizedDate(FormatStyle.FULL).format(to);
+            return String.format("%d. [E][%s] %s(from %s to %s)", this.num, this.getStatusIcon(), this.description, first, end);
+        }
+
+    }
+    public static void main(String[] args) {
+        ArrayList<Task> tasks = new ArrayList<Task>();
+        String currentD = Paths.get("").toAbsolutePath().toString();;
+        Path dirPath = Paths.get(currentD, "data");
+        Path filePath = Paths.get(currentD, "data", "JaneList.txt");
+        //check if data directory exists, and if it doesnt create one
+        try {
+            if (Files.notExists(dirPath)){
+                Files.createDirectory(dirPath);
+            }
+        } catch (IOException err) {
+            System.out.println("Unable to create directory");
+            err.printStackTrace();
+        //Check if the janelist exist inside the folder we have made and if it doesnt, create, if not read it
+        } try {
+            if (Files.notExists(filePath)) {
+                Files.createFile(filePath);
+            } else {
+                List<String> lines = Files.readAllLines(filePath);
+                for (String s:lines) {
+                    //to separate each portion of the task eg D | taskname | deadline to easily see which type of task and deadline
+                    String[] line = s.split("\\|");
+                    int i = Integer.parseInt(line[1]);
+                    boolean b = (i==1);
+                    if (line[0].equals("T")) {
+                        Todo T = new Todo(tasks.size() + 1, line[2]);
+                        T.changeState(b);
+                        tasks.add(T);
+                    } else if (line[0].equals("D")) {
+                        Deadline D = new Deadline(tasks.size() + 1, line[2], LocalDateTime.parse(line[3]));
+                        D.changeState(b);
+                        tasks.add(D);
+                    } else if (line[0].equals("E")) {
+                        Event E = new Event(tasks.size()+1, line[2], LocalDateTime.parse(line[3]), LocalDateTime.parse(line[4]));
+                        E.changeState(b);
+                        tasks.add(E);
+                    }
+
+                }
+
+
+            }
+
+        } catch (IOException err) {
+            System.out.println("Unable to create File");
+            err.printStackTrace();
+        }
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
```

Comment:
I love that you have customized your application!
Perhaps you could also update this logo as well?